engines/director/score.cpp

```diff
@@ -488,6 +498,10 @@ void Score::update() {
 			} else if (tempo == 134) {
 				// Wait for sound channel 2
 				_waitForChannel = 2;
+
+			} else if (tempo >= 136 && tempo <= 183) {
```

Comment:
Digital videos were introduced in D3.1 or so. What does happen in earlier versions of Direcor?